adapters/handlers/rest/configure_api.go

```diff
@@ -957,6 +958,14 @@ func registerModules(appState *state.State) error {
 			Debug("enabled module")
 	}
 
+	if _, ok := enabledModules[moddatabricks.Name]; ok {
```

Comment:
it's a 3rd party module so it can be added to [defaultVectorizers](https://github.com/weaviate/weaviate/blob/main/adapters/handlers/rest/configure_api.go#L779) list